src/cargo/core/resolver/errors.rs

```diff
@@ -140,8 +140,8 @@ pub(super) fn activation_error(
                     msg.push_str("` as well:\n");
                     msg.push_str(&describe_path_in_context(cx, p));
                     msg.push_str("\nOnly one package in the dependency graph may specify the same links value. This helps ensure that only one copy of a native library is linked in the final binary. ");
-                    msg.push_str("Try to adjust your dependencies so that only one package uses the links ='");
-                    msg.push_str(&*dep.package_name());
+                    msg.push_str("Try to adjust your dependencies so that only one package uses the links = '");
+                    msg.push_str(link);
```

Comment:
I would err on the side of removing the first space rather than adding the second.

With the formatting we have today,  having spaces puts more of a burden on the reader to tell when they are dealing with words vs syntax.  Adding backticks could help but only so much.

Looking at `util/toml/mod.rs`, it seems we do a mix of space+backticks and no-space+no-backticks.  A lot of those messages are a lot less dense then this one though.

Reply:
but that's not how anyone actually formats their `Cargo.toml`, right?  we're all writing `links = 'whatever'`?

Reply:
BTW, should we refactor these lines into `format!` macro? It'll marginally increase the readability.

Reply:
Correct but people need to be able to parse this message.

We can either
- Add backticks
- Remove spaces

I lean towards the latter but we should at least do one of these two.

Reply:
actually we're probably all writing `links = "whatever"` with the double quote.

I'll do that and put backticks around this bit, that would be consistent with the rest of the message.  Sound ok?

Reply:
Looking at the messages in practice, the backticks at least help with direct reading and probably ok enough for scanning.